docs/concepts/gas.md

```diff
@@ -0,0 +1,200 @@
+---
+id: gas
+title: Gas
+sidebar_label: Gas
+---
+
+Gas is a dynamic, unified unit of cost of computation and storage on the blockchain as well as network utilization among nodes.  
+
+Accounts automatically buy gas using NEAR tokens at various points in time as when they invoke a function (charged at the moment the transaction is processed) or pay storage rent (charged periodically based on amount of data stored by an account as well as a possible tax on having a short account name to prevent name squatting).
```

Comment:
To be exact, the rent is not paid automatically.  It's accumulating and paid once account executes an incoming transaction or receipt. See:
https://github.com/nearprotocol/nearcore/blob/master/core/primitives/src/account.rs#L18
https://github.com/nearprotocol/nearcore/blob/master/runtime/runtime/src/lib.rs#L258
https://github.com/nearprotocol/nearcore/blob/master/runtime/runtime/src/lib.rs#L570

Reply:
great, thank you

Reply:
will rewrite to this.  is it better?

> Accounts also pay storage rent which accumulates over time and is charged once the account submits a transaction for processing.